scantool/scantool.c

```diff
@@ -640,6 +640,15 @@ l2_do_send(struct diag_l2_conn *d_conn, void *data, size_t len, void *handle)
 
 	msg.len = len;
 	msg.data = (uint8_t *)data;
+
+	if (d_conn->l2proto->diag_l2_protocol == DIAG_L2_PROT_VAG) {
```

Comment:
Yes, the command would be "sendreq 08 02". msg.type is 8, msg.len is 1 and data[0] is 2.